tree/ntuple/v7/test/ntuple_types.cxx

```diff
@@ -886,6 +886,76 @@ TEST(RNTuple, Casting)
    EXPECT_EQ(137, *fieldCast2);
 }
 
+TEST(RNTuple, Float16)
+{
+   FileRaii fileGuard("test_ntuple_float16.root");
+
+   auto f1Fld = RFieldBase::Create("f1", "float").Unwrap();
+   f1Fld->SetColumnRepresentative({ROOT::Experimental::EColumnType::kReal16});
```

Comment:
```suggestion
   f1Fld->SetHalfPrecision();
```

Reply:
Unless `SetHalfPrecision` is added to `RFieldBase`, no? Or, alternatively unless we provide a way to get a reference to the field in such a way that it can be (safely) modified (`model->GetField("f1")` returns a const pointer).